### ISSUE TITLE
Test 243 Failed

### ISSUE THREAD

**Issue by Tenshi65535** [psi4/psi4#2828]
Greetings to all.

I've build the latest v1.7 release successfully but met some problem when running the test suites.
Test 243 failed with the following error:

```bash
243: psi4.driver.p4util.exceptions.TestComparisonError: root
243:     Arrays differ.	_compare_recursive: computed value does not match to atol=1e-06 (o-e: RMS 5.0e-01, MAX 1.6e+00, RMAX 2.0e+00).
243:   Expected:
243:     [[ 0.0291933 -0.3287121  0.7914111 -0.5145527]
243:      [ 0.1791863 -0.7419178  0.1002281  0.6382825]
243:      [ 0.5820757 -0.3705022 -0.5095786 -0.5140483]
243:      [ 0.7926083  0.4519231  0.3224164  0.2521612]]
243:   Observed:
243:     [[-0.029193323165  0.328712055763 -0.791411145833  0.514552749997]
243:      [ 0.179186290535 -0.741917790628  0.100228136947  0.638282528194]
243:      [ 0.582075699497 -0.370502185067 -0.509578634502 -0.514048272222]
243:      [ 0.792608291164  0.451923120902  0.322416398582  0.252161169688]]
243:   Difference (passed elements are zeroed):
243:     [[-0.058386623165  0.657424155763 -1.582822245833  1.029105449997]
243:      [ 0.              0.              0.              0.            ]
243:      [ 0.              0.              0.              0.            ]
243:      [ 0.              0.              0.              0.            ]]
```

It seems the first row is completely negated?

**Comment by JonathonMisiewicz** [comment]
Thanks for the bug report.

What's _happening_ is that test 243 is testing an eigenvector compute routine. The phase is non-deterministic, so the test does postprocessing on the phase. Due to an error in the test itself, the phase postprocessing was not done on the first row, causing the incorrect sign.

tl;dr: Your Psi installation is fine. The test was buggy. Thanks.
